eng/Versions.props

```diff
@@ -32,7 +32,7 @@
     <!-- -->
     <!-- FSharp.Compiler.Service version -->
     <FCSMajorVersion>43</FCSMajorVersion>
```

Comment:
Doesn't this change also justify bumping major version from 43 to 44?

Reply:
Probably not, we haven't changed any major API's or tfm (FCS is ns)